vanilla-framework/vanilla-framework
# Remove docs/divider-list page content

## Done

- Removed [divider-list](https://docs.vanillaframework.io/en/patterns/divider) page content as it's own section.
- Move example to sit under **Lists**
- Next steps (PR) to add into [Lists](https://docs.vanillaframework.io/en/patterns/lists) section as it's more relevant to be in here

## QA

- Pull code
- Run `./docs`
- Open http://0.0.0.0:8104
- Scroll to 'Page structure' and notice it no longer exists

## Details
- Fixes https://github.com/ubuntudesign/design-vanilla-framework/issues/386
